src/ore/src/panic.rs

```diff
@@ -86,7 +87,12 @@ pub fn install_enhanced_handler() {
             return;
         }
 
-        let timestamp = Utc::now().format("%Y-%m-%dT%H:%M:%S%.6fZ").to_string();
+        // can't use if cfg!() here because that will require chrono::Utc import
+        #[cfg(feature = "chrono")]
+        let timestamp = Utc::now().format("%Y-%m-%dT%H:%M:%S%.6fZ ").to_string();
```

Comment:
not sure if this is relevant but it does change the output from two spaces to one space in between the timestamp and the message